glpk/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: glpk
+  version: 4.57
+
+source:
+  fn: glpk-4.57.tar.gz
+  url: http://ftp.gnu.org/gnu/glpk/glpk-4.57.tar.gz
+  md5: 237531a54f73155842f8defe51aedb0f
+
+build:
+  features:
+    - vc9      [win and py27]
+    - vc10     [win and py34]
+    - vc14     [win and py35]
```

Comment:
I mean, if not, you can select just one `vc` feature here (for example, `vc14`) instead of listing them all :-)